package.json

```diff
@@ -68,6 +68,7 @@
     "i18n-js": "^3.8.0",
     "install": "^0.13.0",
     "lodash": "^4.17.21",
+    "lru-cache": "^6.0.0",
```

Comment:
can we try this package?  https://www.npmjs.com/package/react-native-cache

This cache is meant for mobile usage